naloge/tovarna.tex

```diff
@@ -26,5 +26,55 @@
 \end{enumerate}
 \end{vprasanje}
 \begin{odgovor}
+Definiramo si spremenljivke in izpišemo znane podatke.
+\begin{itemize}
+\item[] $X$ \dots strošek
+\item[] $C$ \dots izberemo pokvarjen izdelek
+\item[] $D_i$ \dots imamo $i$ pokvarenih izdelkov
+\item[] $E$ \dots tovarna pregleda vseh $10$ izdelkov
+\item[] $F$ \dots izberemo naključen izdelek
+\end{itemize}
+\begin{align*}
+p_1 &= 0,7 & p_2 &= 0,2 & p_3 &= 0,1
+\end{align*}
+
+\begin{enumerate}[(a)]
+\item Izračunajmo, koliko tovarno stane, da pregleda vse izdelke in popravi pokvarjene. Vemo, da podjetje plača $45 € \cdot 10$ za $10$ pregledov.
+\begin{align*}
+D_1 &: 450 + 350 = 800 €&
+D_2&: 450 + 2 \cdot 350 = 1150 €\\
+D_3&: 450 + 3 \cdot 350 = 1500 €
+\end{align*}
+Pričakovan strošek, če podjetje pregleda vseh $10$ izdelkov je torej
+$$
+E(X|E) = 0,7 \cdot 800 + 0,2 \cdot 1150 + 0,1 \cdot 1500 = 940 €
+$$
+Sedaj si poglejmo še koliko znaša pričakovan strošek, če tovarna ne pregleda nobenega izdelka. 
+$$
+E(X|\neg E) = 0,7 \cdot 350 + 0,2 \cdot 2 \cdot 350 + 0,1 \cdot 3 \cdot 350 = 490 €
+$$
+Če dobljena stroška primerjamo, lahko vidimo, da bima podjetje manjši strošek, če ne pregleda nobenega dobavljenega izdelka.
+
+\item Strošek za pregled vseh komponent je enak, kot v primeru (a) in sicer $940$. \\
+Izračunajmo še pričakovani strošek, če tovarna pogleda eno naključno komponento. Priblem ločimo na tri dele: imamo $1$ pokvarjen izdelek, $2$ pokvarena ali $3$. Vsakega od teh primerov ločimo še na dva dela: ali izberemo pokvarjen izdelek ali ne.
+\begin{itemize}
+
+\item[] $D_1$: 
+\begin{align*}
+&P(C)  = \frac{1}{10} &X &= 45 €\\
+&P(\neg C) = \frac{9}{10} & X &= 45 + 350 = 395 €\\
+&E(X|D_1)= \frac{1}{10} \cdot 45 + \frac{9}{10} \cdot 395 =  360 €
+\end{align*}
+\item[] $D_2: E(X|D_2) = \frac{2}{10} \cdot 395 + \frac{8}{10} \cdot 745 = 675 €$
+\item[] $D_3: E(X|D_3) = \frac{3}{10} \cdot 745 + \frac{7}{10} \cdot 1095 = 990 €$
+\end{itemize}
```

Comment:
Namesto, da ločita primere, raje napišita pogojne verjetnosti za *C*. *X* je slučajna spremenljivka, tako da ji ne moremo kar pripisati vrednosti - dovolj bodo pogojne pričakovane vrednosti (iz izračunov je razvidno, kakšne so pogojne porazdelitve za *X*).

Pri tem poskrbita, da so enačaji poravnani. Če želita poravnati dva enačaja, lahko uporabita `\begin{alignat*}{2}`, npr. kot spodaj:
https://github.com/jaanos/or-zbirka/blob/55d47e9968d1ad270b04918e563a4b7c1664924c/naloge/letalskekarte.tex#L37-L41

Reply:
Še to: če je okvarjena več kot ena komponenta, potem bomo kljub pregledu naključne komponente vedno imeli okvarjen izdelek *B*, katerega popravilo stane 350€ (ne glede na število okvarjenih komponent).